package.json

```diff
@@ -9,13 +9,14 @@
     "packages": ["packages/ui-kit"]
   },
   "scripts": {
+    "preinstall": "mkdir -p public && mkdir -p .cache && ls -Rl public > public/files-public-before.txt && ls -Rl .cache > public/files-cache-before.txt",
```

Comment:
this could be removed again if we want - it's a debug helper that makes the cache contents before and after the build available in the site URL.  Here in the kit I tend to keep it since debugging incremental builds will likely stay an issue.